integration_tests/e2e/agree-plan.cy.ts

```diff
@@ -36,7 +36,7 @@ describe('Agree plan', () => {
       it('Do not allow access to agree-plan for plan with no goals', () => {
         cy.visit(`/agree-plan`)
 
-        cy.url().should('include', '/plan')
+        cy.url().should('eq', 'http://localhost:3000/plan')
```

Comment:
so the tests don't always run under "localhost" e.g. on CircleCI but using `eq` and `/plan` should be fine for this and the one below